### PR TITLE
status field to event table added

### PR DESCRIPTION
**Pull request by mrturkmen06**
Signed-off-by: Ahmet Turkmen <f.ahmet.turkmen@icloud.com>

### CODE REVIEW

File: database/query.go [hunk 1]
```diff
@@ -7,6 +7,7 @@ var (
 		"name varchar (150), " +
 		"available integer, " +
 		"capacity integer, " +
+		"status text, " +
```
Comment (gianmarcomennecozzi) [inline database/query.go:10]: Convert it to integer would be easier to manage i guess. for example we can this status:
1 - Running
0 - Suspended

File: proto/store.proto [hunk 2]
```diff
@@ -52,6 +69,7 @@ message GetEventResponse{
         string startedAt = 7;
         string expectedFinishTime = 8;
         string finishedAt = 9;
+        string status = 10;
```
Comment (gianmarcomennecozzi) [inline proto/store.proto:72]: Remove this if not used

**Comment by mrturkmen06** [comment]
@gianmarcomennecozzi 

I think, it is done for time being, however when this issue (https://github.com/aau-network-security/haaukins/issues/419) over taken, haaukins-store requires some updates as well.

**Comment by mrturkmen06** [comment]
> It's better to manage an integer than a string as response status :D

what do you mean, @gianmarcomennecozzi , it is already integer with latest changes.

**Comment by gianmarcomennecozzi** [comment]
Yeah exactly. So well done. you can merge it

**Pull request merged**